wiki/Chat_Console/th.md

```diff
@@ -1,102 +1,103 @@
-# หน้าต่างพูดคุยอย่างง่าย
+# หน้าต่างพูดคุย
 
-กด `F8` จากที่ไหนก็ได้ในเกม หรือกดปุ่ม `Show Chat` ที่อยู่มุมล่างขวาเพื่อแสดงหน้าต่างแชทที่มีขนาดหนึ่งในสามของหน้าจอ
+กด `F8` จากเกือบทุกที่ในเกม หรือกดปุ่ม `Show Chat` ที่อยู่มุมล่างขวา เพื่อแสดงหน้าต่างแชทที่มีขนาดหนึ่งในสามของหน้าจอ
 
-![Chat Console](img/Chatconsole1.png "Chat Console")
+![หน้าต่างแชท](img/Chatconsole1.png "หน้าต่างแชท")
 
-- แท็บที่เห็นจะแสดงช่องที่คุณสามารถเข้าไปดูได้ แค่กดที่แท็บก็เปลี่ยนช่องได้แล้ว หรืออาจจะกดปุ่ม `+` สีเหลืองเพื่อเลือกช่องใหม่ที่ต้องการ
-- สีของผู้เล่นต่างๆ มีความหมายต่างกัน
+- แท็บที่เห็นจะแสดงช่องที่คุณสามารถเข้าไปดูได้ แค่กดที่แท็บก็เปลี่ยนช่องได้แล้ว กดปุ่ม `New` เพิื่อเปิดรายชื่อช่องสำหรับเข้าร่วม
+- สีของผู้เล่นต่าง ๆ มีความหมายต่างกัน
 
 | สี | คนนั้นเป็นใคร |
 | :-- | :-- |
-| **ขาว** | **คุณ**นั่นแหละ |
-| **เหลืองอ่อน** | ไม่ใช่ Supporter |
-| **เหลือง** (เข้มกว่าข้างบน) | [osu!supporter](/wiki/osu!supporter) |
+| **ขาว** | คุณ |
+| **เหลืองอ่อน** | ผู้เล่นที่ไม่ใช่ Supporter |
+| **เหลืองเข้ม** | [osu!supporter](/wiki/osu!supporter) |
```

Comment:
```suggestion
| **เหลืองอ่อน** | ผู้เล่นที่ไม่ใช่ผู้สนับสนุน|
| **เหลืองเข้ม** | [osu!ผู้สนับสนุน](/wiki/osu!supporter) |
```
Translated to "osu!ผู้สนับสนุน" according to the website. 
It really sounds weird. Maybe I will change in crowdin to "osu!supporter" stick with the Thai name for now tho